### DISCUSSION TITLE
Avatar is animated

### DISCUSSION THREAD

**Discussion by rogerjowett** [community/community#75239] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My avatar is animated why doesn't github animated my avatar please

**Comment by sky9262** [comment]
The reason for this is that GitHub doesn't support animated avatars. 
An avatar is like a small picture that represents you on the website. When you say your avatar is animated, it means it has moving pictures or "animations" in it, like a GIF or a video. 
But GitHub doesn't have the ability to show those moving pictures, so your avatar appears as a regular, still picture instead. GitHub is designed more for sharing code and projects, so it doesn't prioritize having animated avatars. Not all websites can do everything we might want, so we have to understand the limitations of each website and adjust our expectations accordingly.

**Comment by rogerjowett** [comment]
Yeah but neither does Google Gmail or youtube

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @rogerjowett! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, we don't currently support animated GIFs for profile pictures but we do actually have a feature request here for that capability: https://github.com/orgs/community/discussions/46598.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by rogerjowett** [comment]
Why is it impossible to communicate with github programmers?
